feature-wallet-impl/src/main/java/jp/co/soramitsu/wallet/impl/presentation/cross_chain/setup/CrossChainSetupViewModel.kt

```diff
@@ -566,9 +577,10 @@ class CrossChainSetupViewModel @Inject constructor(
     }
 
     override fun onHistoryClick() {
-        originChainId?.let {
+        destinationChainId?.let {
```

Comment:
If the destinationChainId is null then nothing will happen? I think we have to handle this case

Reply:
It will never happen. But we can send error stack trace to developers, if we have some system for remote logging. What do you think about how we need to handle error here?

Reply:
Send stack trace to the developers sounds good, but we don't have a feature like this now. I will write down this suggestion and discuss with leads. Ok, let's keep this function as it is.